Restore and cleanup docker compose files (#1057)
indigo-iam/iam: compose/trust-anchors/README.md

> # Trust-anchor container

Reviewer: ```suggestion
# Trust-anchors container
```

> This folder holds a trustanchor container which populates volumes

Reviewer: ```suggestion
This folder holds a container of trust anchors which populates volumes
```

> that can be mounted in the other services such to establish a trust 

Reviewer: ```suggestion
that can be mounted in the other services to establish a trust 
```

framework based on X.509 certificates created on-the-fly.

> The trustanchor container populates the following volumes

Reviewer: ```suggestion
The trust-anchors container populates the following volumes
```
